src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");

```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. You could say 'Excavator has stopped its work.' to clearly indicate which Machine has stopped working.